src/Contract/TextEnvelope.purs

```diff
@@ -0,0 +1,25 @@
+module Contract.TextEnvelope
+  ( module TextEnvelope
+  , textEnvelopeBytes
+  ) where
+
+import Contract.Prelude
+
+import Cardano.TextEnvelope (textEnvelopeBytes, TextEnvelopeType) as TE
```

Comment:
Why have two qualified names?

Reply:
One for re-exporting, the other is used to define `textEnvelopeBytes` in `Contract`. Not having it qualified would cause name ambiguity, and adding it to the other import would cause a collision on export